tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -101,6 +101,20 @@ public static class Tags
         /// </summary>
         public const string SqlRows = "sql.rows";
 
+        /// <summary>
+        /// The service name of a remote service.
+        /// </summary>
+        public const string PeerService = "peer.service";
+
+        /// <summary>
+        /// The name of the attribute that determined the peer.service tag value. Expected values are:
+        /// <ul>
+        ///   <li>{source_attribute} when the tag was set to a default value, using a defined precursor attribute</li>
+        ///   <li>peer.service when the tag was set by the user</li>
+        /// </ul>
+        /// </summary>
+        public const string PeerServiceSource = "_dd.peer.service.source";
```

Comment:
I don't think this should be public?

```suggestion
        internal const string PeerServiceSource = "_dd.peer.service.source";
```

Reply:
Done in 93727dffab6149900972552d868cfc3cef76d472